rbac/management/role/definer.py

```diff
@@ -93,6 +93,8 @@ def _make_role(data, dual_write_handler):
             role.access.all().delete()
             role_obj_change_notification_handler(role, "updated")
         else:
+            if force_create_relationships:
+                dual_write_handler.replicate_new_system_role(role)
```

Comment:
Maybe can we add here some log message also ? I am not sure if we should keep this in case: 
```
 logger.info("No change in system role %s", name)
```

Reply:
I can add a log message something like `logger.info("Replicated system role %s", name)`? I guess we could also return role in the same block to skip the no change log message?

Reply:
yes, it sounds great